pandas/core/api.py

```diff
@@ -24,8 +24,8 @@
 from pandas.core.panel import Panel, WidePanel
 from pandas.core.panel4d import Panel4D
 from pandas.core.reshape.reshape import (
```

Comment:
see if you can move these reshape imports here to pandas.core.reshape.api (with the other ones); reshape.api is already imported to ``pandas.__init__``, just make more sense there